### PR TITLE
Feature/7 timeorder helpers

### PR DESCRIPTION
**Pull request by ray-ray**

### CODE REVIEW

**Comment by ray-ray** [comment]
[jhhalchemy time_order helper](https://trello.com/c/X1wIH7Ws/28-jhhalchemy-timeorder-helper)

**Comment by ray-ray** [comment]
[jhhalchemy time_order helper](https://trello.com/c/X1wIH7Ws/28-jhhalchemy-timeorder-helper)

File: jhhalchemy/model/time_order.py [hunk 1]
```diff
@@ -52,3 +52,48 @@ def read_time_range(cls, *args, **kwargs):
...
+    :param model_cls: the class of the model to return
+    :param args: arguments specific to the model class
+    :param kwargs: start_timestamp and end_timestamp (see below) as well as keyword args specific to the model class
+    :keyword start_timestamp: the most recent models set before this and all after, defaults to 0
+    :keyword end_timestamp: only models set before (and including) this timestamp, defaults to now
+    :return: model generator
+    """
+    start_timestamp = kwargs.get('start_timestamp')
+    end_timestamp = kwargs.get('end_timestamp')
+    if (start_timestamp is not None) and (end_timestamp is not None) and (start_timestamp > end_timestamp):
```
Comment (imjohsep) [inline jhhalchemy/model/time_order.py:84]: Does `>=` also constitute an InvalidTimestampRange?

**Review by imjohsep — APPROVED** [review]

File: jhhalchemy/model/time_order.py (hunk 1 above)
Comment (ray-ray) [reply jhhalchemy/model/time_order.py:84]: No because the endpoints are inclusive, so start == end would be the row with that exact time.

**Pull request merged**